.travis.yml

```diff
@@ -21,9 +21,14 @@ before_deploy:
 after_deploy:
   - docker logout
 
+addons:
+  apt:
+    packages:
+    - libgconf-2-4
```

Comment:
what is this about?

Reply:
ok, I see. Maybe add the issue as a comment there?

Reply:
github comment?

Reply:
smth like
```yaml
# cypress test fails without it
# see https://github.com/cypress-io/cypress/issues/4069 
addons:
  apt:
    packages:
    - libgconf-2-4
```

Reply:
db59654